controllers/kindergarten-controller.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="de-DE">
+<head>
+    <meta charset="UTF-8">
+    <title>Über uns</title>
```

Comment:
```suggestion
    <title>Kindergarten</title>
```
Der Titel stimmt nicht.